backend/src/data/repositories/userOrganizationRepositry.ts

```diff
@@ -53,6 +54,22 @@ class UserOrganizationRepository extends Repository<UserOrganization> {
     return users;
   }
 
+  async getUserOrganization(organizationId: string, userId: string): Promise<IUserOrganization> {
+    const userOrganization = await this.findOne({ where: { userId, organizationId } });
+    if (!userOrganization) {
+      throw new CustomError('User organization not found.', 404);
+    }
+    const { id } = userOrganization;
+    const response = await this.createQueryBuilder()
+      .select(this.select)
+      .from(UserOrganization, 'user_organization')
+      .where('user_organization.id = :id', { id })
+      .leftJoin('user_organization.user', 'user')
+      .getOne();
```

Comment:
it's unnecessary to set the from clause if you're creating this query in the UserOrganizationRepository
```suggestion
    const response = await this.createQueryBuilder('userOrganization')
      .select(this.select)
      .where('userOrganization.id = :id', { id })
      .leftJoin('userOrganization.user', 'user')
      .getOne();
```

Reply:
I've removed this line but after that it broke and I couldn't get userOrganization